google-books/src/services/bookProcessor.js

```diff
@@ -0,0 +1,50 @@
+export const bookProcessor = (book) => {
+	const title = book.volumeInfo.title ? book.volumeInfo.title : "No Title";
```

Comment:
you can think of handling too long titles like this:

`  if (title.length > 200) {
        const indexToTrim = title.indexOf(" ", 50);
        title = title.slice(0, indexToTrim) + "...";
    }`